app/controllers/api/v1/env_controller.rb

```diff
@@ -26,6 +26,7 @@ class EnvController < ApiController
       def index
         render_data data: {
           OPENID_CONNECT: ENV['OPENID_CONNECT_ISSUER'].present?,
```

Comment:
We can remove this now no?

Reply:
Currently it would be unused, yes. Not sure if there could be a future use for this though.

Reply:
I would remove it then if possible please

Reply:
Done.